### PR TITLE
Laser usage1

### PR DESCRIPTION
**Pull request by nicolasdb**
ok, laser badge USE1
Qu'est-ce qu'on pense de ça?

### CODE REVIEW

File: badge-src/badge-lasersaur.md [hunk 1]
```diff
@@ -1,21 +1,57 @@
...
-[todo] creating a scale for leveling up. like 
-- level 1 = 1 h, 
-- level 2 = 2 h, 
-- level 3 = 4 h, 
-- level 4 = 8 h, 
-- level 5 = 16 h,
-- level 6 = 32 h,
-- 
+### unlock level 1 from completing one of these challenges:
+- [x] [Making a box](https://nicolasdb.github.io/post/boxes/) to store tools and shit.
```
Comment (satblip) [inline badge-src/badge-lasersaur.md]: love it ! Qui procure les matériaux ?

Comment (nicolasdb) [reply badge-src/badge-lasersaur.md]: OF, j'ai acheté du stock et il y a toutes les chutes

**Comment by DewiBrunet** [comment]
So cool!
I will have a few precisions to add once I find the way to do it #githubnoob (like the mekano specialisation, specific challenge with the mini A5, bonus xp for mentoring, knowing how to organise the bank of waste...)

Questions : 
- don't we need a few translation in french? (keywords like focus, feed and power)
- how to deliver it? only on github or with a paper checklist to add to the passeport?

**Comment by nicolasdb** [comment]
nhaaa, no translation, especially for these keywords, we should be able to explain them. 
Since we have to explain what is the focus, the feed (speed) and power (%)
Either way, softwares are in english. And you must know what setting you're looking for. 
_The passport and badges aim to go abroad, and not only in France ^^_

but, we can explain in french orally during the introduction. that's no problem, right?

- yup, we definitely need something more than just the stickers. But these should be connected to openbadge system soon. 
Let's think about something in between.

**Review by satblip — APPROVED** [review]

**Comment by nicolasdb** [comment]
Je vais donc tester la doc de mon nouveau badge

**Pull request merged**